src/layouts/UseCases.tsx

```diff
@@ -1,3 +1,448 @@
+    minHeight={{ base: "800px", lg: "608px" }}
+    maxHeight={{ base: "full", lg: "608px" }}
+    width="full"
+    position="relative"
+    {...props}
+  />
+)
+
+const TitleCard = (props: ChildOnlyProp) => {
+  const boxShadow = useToken("colors", "cardBoxShadow")
+
+  return (
+    <Flex
+      bg={{ base: "ednBackground", lg: "background.base" }}
+      border="1px"
+      borderColor="border"
+      borderRadius={{ base: "none", lg: "base" }}
+      boxShadow={{ lg: boxShadow }}
+      flexDirection="column"
+      maxWidth={{ base: "full", lg: "container.sm" }}
+      zIndex="docked"
+      p={8}
+      position="absolute"
+      top={{ base: "unset", lg: 24 }}
+      left={{ base: 0, lg: 24 }}
+      bottom={{ base: 0, lg: "unset" }}
+      right={{ base: 0, lg: "unset" }}
+      {...props}
+    />
+  )
+}
+
+const Title = (props: ChildOnlyProp) => <Heading1 mt={4} {...props} />
+
+const Page = (props: FlexProps) => (
+  <Flex
+    flexDirection={{ base: "column", lg: "row" }}
+    justifyContent="space-between"
+    mx="auto"
+    mb={16}
+    pt={{ lg: 16 }}
+    width="full"
+    sx={{ "h2:first-of-type": { mt: { lg: 0 } } }}
+    {...props}
+  />
+)
+
+const InfoColumn = (props: ChildOnlyProp) => (
+  <Flex
+    as="aside"
+    flexDirection="column"
+    flex="0 1 400px"
+    ml={8}
+    mr={16}
+    position="sticky"
+    top="6.25rem"
+    height={calc("100vh").subtract("80px").toString()}
+    {...props}
+  />
+)
+
+const InfoTitle = (props: ChildOnlyProp) => (
+  <Heading2
+    fontSize={{ base: "2.5rem", lg: "5xl" }}
+    textAlign={{ base: "left", lg: "right" }}
+    mt={0}
+    {...props}
+  />
+)
+
+const StyledButtonDropdown = ({
+  list,
+  ...rest
+}: FlexProps & Pick<ButtonDropdownProps, "list">) => (
+  <Flex align="flex-end" justify="flex-end" mb={8} {...rest}>
+    <ButtonDropdown list={list} w={{ base: "full", lg: "auto" }} minW="240px" />
+  </Flex>
+)
+
+const MobileButtonDropdown = (
+  props: ComponentProps<typeof StyledButtonDropdown>
+) => <StyledButtonDropdown mb={0} {...props} />
+
+const ContentContainer = (props: Pick<BoxProps, "id" | "children">) => {
+  const lgBp = useToken("breakpoints", "lg")
+
+  return (
+    <Box
+      as="article"
+      flex={`1 1 ${lgBp}`}
+      position="relative"
+      px={8}
+      pb={8}
+      {...props}
+      sx={{
+        ".featured": {
+          pl: 4,
+          ml: -4,
+          borderLeft: "1px dotted",
+          borderColor: "primary.base",
+        },
+        ".citation p": {
+          color: "text200",
+        },
+      }}
+    />
+  )
+}
+
+const MobileButton = (props: ChildOnlyProp) => {
+  const borderColor = useToken("colors", "border")
+  return (
+    <Box
+      bg="background.base"
+      boxShadow={`0 -1px 0 ${borderColor}`}
+      position="sticky"
+      bottom={0}
+      zIndex={99}
+      p={8}
+      width="full"
+      {...props}
+    />
+  )
+}
+
+export const UseCasesLayout = ({ children, frontmatter, slug, tocItems }) => {
```

Comment:
Lets define the types for the layout

Reply:
We have `frontmatter, slug, tocItems` defined locally inside `md.ts` right now as `type Items`... I'd suggest we rename to `interface PageContent` and pull that into `@/lib/interfaces` so we can reuse in places like right here... will go ahead with that, but let me know if you disagree